EIPS/eip-2935.md

```diff
@@ -177,7 +177,7 @@ This address is currently exempt from [EIP-158](./eip-158.md) cleanup in Kaustin
 
 ### Gas costs and witnesses
 
-Since now `BLOCKHASH` is served from state, the clients now **additionally** charge the corresponding warm or cold `SLOAD` costs. For verkle based networks this would imply doing and bundling corresponding accesses (and gas charges) of `SLOAD`.
+Since now `BLOCKHASH` is served from state, the clients now **additionally** charge the corresponding warm or cold `SLOAD` costs. For verkle based networks this would imply doing and bundling corresponding accesses (and gas charges) of `SLOAD`. However, the corresponding code access charges (if implemented via executing the evm contract code) are not charged.
```

Comment:
```suggestion
Since now `BLOCKHASH` is served from state, the clients now **additionally** charge the corresponding warm or cold `SLOAD` costs. For verkle based networks this would imply doing and bundling corresponding accesses (and gas charges) of `SLOAD`. 
However, the corresponding contract code accesses (and  charges)  are not levied to keep it simple so that the clients can choose to directly implement rather than EVM contract `get` execution.
```